website/client/src/router/index.js

```diff
@@ -338,6 +342,22 @@ const router = new VueRouter({
         { name: 'contributors', path: 'contributors', component: HeroesPage },
       ],
     },
+
+    {
+      name: 'adminPanel',
+      path: '/admin-panel',
+      component: AdminPanelPage,
+      meta: { requiresAdmin: true },
+      children: [
```

Comment:
if you Save and Clear data after loading an user in the admin panel the url doesn't change and clicking the Load User button again it refreshes the page

Reply:
So sorry, I only just noticed these review comments. I must have missed the email. :(

Yes, that behaviour is intentional.
"Save and Clear" for the contributor section is necessary because increasing the Tier will change other parts of the user (Gems, items, pets) and so you want to ensure that none of the old data remains.

However the User ID is kept in the search box and the URL doesn't change so that you can easily re-fetch the data for the same user, in case you want to check their new settings or make other changes to their account.

The reload is necessary because the vue router can't handle "moving" from one URL to the same URL. The same thing happens if you load a user and then click the "Load User" button a second time (e.g., if you want to back out of any changes you made). That's this code: https://github.com/HabitRPG/habitica/pull/12652/files#diff-7a3e28ebec99ac2c419cdc471d62b162R60-R75

I'm sure there's a nicer way to do all this but I messed around with it for hours and this was the best option that covered all cases. If it was for the general user population it would be worth spending more time on it, but I'm not sure it's worth it for only the staff and mods.

Reply:
> However the User ID is kept in the search box and the URL doesn't change so that you can easily re-fetch the data for the same user, in case you want to check their new settings or make other changes to their account.

I'd like this to be changed, I'm fine with keeping the user id in the search box but if we go back to the main page the url should be changed accordingly

> "Save and Clear" for the contributor section is necessary because increasing the Tier will change other parts of the user (Gems, items, pets) and so you want to ensure that none of the old data remains.

I'm fine if we want to keep the Save and Clear but I'm pretty sure the api route used to update the data returns everything for that user so you could take the response and Object.assign(this.hero, response) to update it (but if it's too much hassle, Save and clear is totally fine for me)